signature/policy_eval_signedby_test.go

```diff
@@ -123,7 +123,7 @@ func TestPRSignedByIsSignatureAuthorAccepted(t *testing.T) {
 	// A signature which does not GPG verify
 	pr, err = NewPRSignedByKeyPath(ktGPG, "fixtures/public-key.gpg", prm)
 	require.NoError(t, err)
-	// Pass a nil pointer to, kind of, test that the return value does not depend on the image parmater..
+	// Pass a nil pointer to, kind of, test that the return value does not depend on the image parameter..
```

Comment:
Not sure about the `..` ...

(I can change it if requested)

Reply:
Looking at the other `, kind of,` comments around, that was intended to be just a sentence-ending period.